### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by triclambert**

### CODE REVIEW

File: docs/guides/contributor/workflows.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ But at this point, you have certainly entered the "I know what I am doing" stage
 
 ## Run the `test-remote` suite against a remote cluster
 
-See [the `test-remote` suite of tests](./test-remote), a document dedicated to this test suite.
+See [the `test-remote` suite of tests](https://github.com/opstrace/opstrace/tree/main/test/test-remote), a document dedicated to this test suite.
```
Comment (triclambert) [inline docs/guides/contributor/workflows.md:101]: @jgehrcke thoughts on this? Would you rather link directly to the README.md?

File: docs/references/roadmap.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <!-- markdownlint-disable MD036 -->
 
-Our early access version provides a foundation that solves many [real pain points](/blog/public-launch-announcement) that people have today.
+Our early access version provides a foundation that solves many [real pain points](https://opstrace.com/blog/public-launch-announcement) that people have today.
```
Comment (triclambert) [inline docs/references/roadmap.md:5]: Not in love with this, but it's better than being broken.

File: docs/guides/contributor/workflows.md (hunk 1 above)
Comment (jgehrcke) [reply docs/guides/contributor/workflows.md:101]: This is fine / better than linking right to the README. Thanks!

**Comment by jgehrcke** [comment]
Cool, thanks :)

**Pull request merged**